scipy/special/xsf/binom.h

```diff
@@ -82,8 +82,6 @@ XSF_HOST_DEVICE inline double binom(double n, double k) {
     return 1 / (n + 1) / cephes::beta(1 + n - k, 1 + k);
 }
 
-XSF_HOST_DEVICE inline float binom(float n, float k) {
-    return binom(static_cast<double>(n), static_cast<double>(k));
-}
+XSF_HOST_DEVICE inline float binom(float n, float k) { return binom(static_cast<double>(n), static_cast<double>(k)); }
```

Comment:
I'd prefer to leave single-line function body as is than to merging it in the same line as function signature, so that if we add more lines into the function body later, we don't have to "undo" the merge and cause spurious diff.

Reply:
Do you know how to change the config file to get what you are looking for?

Reply:
Maybe https://clang.llvm.org/docs/ClangFormatStyleOptions.html#allowshortfunctionsonasingleline

Reply:
@fancidev do you prefer the diff after https://github.com/scipy/scipy/compare/11188fd5fd20877880fd15d23cddd3631fa4101e..791fc539b3f9324cd6bdbf0329905edc4650240a ?

Reply:
Thanks for checking. It seems the new option would convert an existing single-line function body into multiple lines. I feel this is unnecessary if the function body is indeed a simple wrapper.

It would have been best if clang-format provided a "preserve" option that leaves single-line/multi-line function body unchanged, but seems there's no such option. I think I like the previous behavior better (i.e. merging short one-liner into the line of `{`) as it creates fewer "bad" choices (according to my preference) than the new option.